packages/terra-core-docs/src/terra-dev-site/doc/form-input/InputField.2.doc.mdx

```diff
@@ -4,6 +4,7 @@ import InputFieldExample from './example/InputField?dev-site-example';
 import RequiredInputFieldExample from './example/RequiredInputField?dev-site-example';
 import IncompleteInputFieldExample from './example/IncompleteInputField?dev-site-example';
 import NumberInputFieldExample from './example/NumberInputField?dev-site-example';
+import PlaceholderInputFieldExample from './example/PlaceholderInputField?dev-site-example';
```

Comment:
same here !! we need delete these examples as well

Reply:
Removed the example.